index.html

```diff
@@ -68,9 +85,10 @@ <h5>Multi-Post Stories</h5>
             <li>Bootstrap</li>
```

Comment:
- [ ] [REQUIRED] As mentioned by the previous reviewer, please the border for the list of technologies uses this `#8993A4` color and not `black`. Kindly change that so it fits with that of the Figma design. 

### Your design 
![image](https://user-images.githubusercontent.com/84156714/193331702-fcaf08f6-5c50-49c3-afb5-a0957159c024.png)

### Figma Design
![image](https://user-images.githubusercontent.com/84156714/193331908-76d19af7-c2e9-4a58-9a5a-f78c9210d8f2.png)